Project.toml

```diff
@@ -22,9 +22,11 @@ Symbolics = "0c5d862f-8b57-4792-8d23-62f2024744c7"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [weakdeps]
+BifurcationKit = "0f109fa4-8a5d-4b75-95aa-f515264e7665"
 HomotopyContinuation = "f213a82b-91d6-5c5d-acf7-10f1c761b327"
 
 [extensions]
+CatalystBifurcationKitExtension = "BifurcationKit"
 CatalystHomotopyContinuationExtension = "HomotopyContinuation"
 
 [compat]
```

Comment:
I think we need compat entries for each weakdep too. See https://github.com/SciML/SciMLBase.jl/blob/master/Project.toml

Reply:
Added:
```
BifurcationKit = "0.3"

...

HomotopyContinuation = "2.9"
```